adafruit_tca8418.py

```diff
@@ -233,7 +245,7 @@ def _get_gpio_register(self, reg_base_addr, pin_number):
         reg_base_addr += pin_number // 8
         return self._get_reg_bit(reg_base_addr, pin_number % 8)
 
-    def get_pin(self, pin):
+    def get_pin(self, pin: int) -> digitalio.DigitalInOut:
```

Comment:
Due to the fact that a `DigitalInOut` is defined in this module, you should make this return the forward reference `"DigitalInOut"`.